packages/docusaurus-theme-live-codeblock/src/theme/Playground/index.js

```diff
@@ -11,13 +11,55 @@ import clsx from 'clsx';
 
 import styles from './styles.module.css';
 
-function Playground({children, theme, transformCode, ...props}) {
+const typescriptExtensions = ['ts', 'tsx'];
+const transformJs = (code) => `${code};`;
+
+function Playground({children, theme, className, scope}) {
+  const [tsTranspileError, setTsTranspileError] = React.useState(null);
+  const transformTs = React.useCallback((code) => {
+    try {
+      // eslint-disable-next-line global-require
+      const {code: transformed} = require('@babel/standalone').transform(code, {
```

Comment:
`require`d inside here in the hope that it will dynamically import it only if there is a code block written in Typescript... but I'm not fully sure if this hack works.

Reply:
seems like it does work?